evalai-cli/bin/activate

```diff
@@ -0,0 +1,78 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
@nikhilmudholkar 
Thanks for the PR!
There are many unnecessary additions which is not required in this PR
It seems you have cloned the project in the root of `virtualenv` or `venv`.
Only change in `docs/index.html` is required, please revert unnecessary additions.

Reply:
@vkartik97 due to some complications I have submitted another PR for the same issue.  Fortunately, all the tests are passing for it. Can you please merge it. Thanks a lot